filament/src/MaterialParser.h

```diff
@@ -50,11 +50,9 @@ class MaterialParser {
 
     bool parse() noexcept;
     bool isShadingMaterial() const noexcept;
```

Comment:
We probably don't need this one anymore then? We should also have a getDomain() if we don't already

Reply:
Good point, yeah it can probably be removed. I'll add a `getDomain` too.

Reply:
Oh, just realized I already added a getter: `getMaterialDomain` on line 73.